src/services/bundle.service.js

```diff
@@ -161,16 +160,36 @@ async function uploadResourcesFromBundle(entries, headers, baseUrl, baseVersion,
     entryHeaders['X-Provenance'] = headers['x-provenance'];
   }
   const requestsArray = scrubbedEntries.map(async entry => {
-    const { url, method } = entry.request;
-    const destinationUrl = `${protocol}://${path.join(headers.host, baseUrl, baseVersion, url)}`;
-    return axios[method.toLowerCase()](destinationUrl, entry.resource, {
-      headers: entryHeaders
-    }).catch(e => {
+    const { method } = entry.request;
+
+    return replaceAxiosWithMongo(entry, method).catch(e => {
       return e.response;
     });
   });
   const requestResults = await Promise.all(requestsArray);
   return requestResults;
 }
 
+async function replaceAxiosWithMongo(entry, method) {
```

Comment:
I would change this function name to something like `insertBundleResources` or something. I think replaceAxiosWithMongo will be confusing outside of this task.